lib/server/serve.js

```diff
@@ -0,0 +1,6 @@
+'use strict'
```

Comment:
@p16 I had actually called it server on purpose, sever/serve.js looks cleaner that `server/server.js`